angular/openmrs-search/openmrs-search.html

```diff
@@ -0,0 +1,103 @@
+<div ng-show="vm.columnsDetails && vm.data.length > 0 && !isUserTyping && vm.query.length > 0">
+    <input id="includeVerbose" type="checkbox" ng-model="vm.enableDetails">
+    <label for="includeVerbose">Show Details</label>
+</div>
+
+<div class="search-input" align="center">
+    <input class="field-display ui-autocomplete-input" id="inputNode" type="text" ng-model="vm.query"
+           ng-change="vm.timeoutRefresh()">
+    <button id="searchButton" name="searchButton" ng-click="vm.getData()">Search</button>
+</div>
+
+<div align="center">
+    <span ng-show="vm.notificationEmptyQuery()">Type query to search data</span>
+</div>
+<div align="center">
+    <span ng-show="vm.notificationNoEntriesFound()">No entries found for <b>{{vm.query}}</b></span>
+</div>
+
+<div align="center" ng-show="vm.notificationLoading()">
+    <img ng-src="{{vm.logo}}" width="64" height="64">
+    <br>
+    <span><b>Loading...</b></span>
+</div>
+
+<!--TABLE TYPE-->
+<table ng-show="vm.showTable()">
```

Comment:
I think it would be better if I add some component attribute to openmrs-list like "searchPanel = "true", which will pimp openmrs-list component into openmrs-search, but it would work on same template and controller, which would be more viable.